charts/substra-backend/templates/job-migrations.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
     {{ include "substra.labels" . | nindent 4 }}
     app.kubernetes.io/name: {{ template "substra.fullname" . }}-migrations
   annotations:
-    "helm.sh/hook": post-install,post-upgrade
```

Comment:
I'm not very familiar with helm hook, but are we changing the triggers for the migration job?

Reply:
Before, we had this job (which did migrations) +this job applied after the deployment stabilised which did migration + user/organization management. This PR remove the init container doing migration in the server, and trigger the job when the deployment is started

Reply:
Sorry I'm not sure I follow which jobs are which when you say "this job" and "this job". Afaik the migration job is the only one job we have.